plugins/vpp/ifplugin/vppcalls/vpp2001_379/gtpu_vppcalls.go

```diff
@@ -0,0 +1,79 @@
+package vpp2001_379
+
+import (
+	"errors"
+	"net"
+
+	ifs "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+	vpp_gtpu "github.com/ligato/vpp-agent/plugins/vpp/binapi/vpp2001_379/gtpu"
+)
+
+func (h *InterfaceVppHandler) gtpuAddDelTunnel(isAdd uint8, gtpuLink *ifs.GtpuLink, vrf uint32, multicastIf uint32) (uint32, error) {
+	req := &vpp_gtpu.GtpuAddDelTunnel{
+        IsAdd:          isAdd,
+		McastSwIfIndex: multicastIf,
+        //EncapVrfID:     vrf,
+        EncapVrfID:     gtpuLink.EncapVrfId,
+		Teid:           gtpuLink.Teid,
+    }
+
+    if gtpuLink.DecapNext == ifs.GtpuLink_DEFAULT {
+        req.DecapNextIndex = 0xFFFFFFFF
+    } else {
+        req.DecapNextIndex = uint32(gtpuLink.DecapNext)
+    }
+
+	srcAddr := net.ParseIP(gtpuLink.SrcAddr)
+	if srcAddr == nil {
+		err := errors.New("bad source address for GTPU tunnel")
+		return 0, err
+	}
+
+	dstAddr := net.ParseIP(gtpuLink.DstAddr)
+	if dstAddr == nil {
+		err := errors.New("bad destination address for GTPU tunnel")
+		return 0, err
+	}
+
+	if gtpuLink.SrcAddr == gtpuLink.DstAddr {
+		err := errors.New("source and destination are the same")
+		return 0, err
+	}
+
+    if srcAddr.To4() != nil && dstAddr.To4() != nil {
+        req.IsIPv6 = 0
+        req.SrcAddress = []byte(srcAddr.To4())
+        req.DstAddress = []byte(dstAddr.To4())
+    } else if srcAddr.To16() != nil && dstAddr.To16() != nil {
+        req.IsIPv6 = 1
+        req.SrcAddress = []byte(srcAddr.To16())
+        req.DstAddress = []byte(dstAddr.To16())
```

Comment:
I wonder if this is auto-generated or not. (The weird package name suggests it is.)

We could overlook the weird indentation for an auto-generated file, otherwise I'd suggest to fix it.